src/test/java/com/xceptance/neodymium/tests/BrowserstackTest.java

```diff
@@ -0,0 +1,45 @@
+package com.xceptance.neodymium.tests;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.runner.JUnitCore;
+import org.junit.runner.Result;
+import com.xceptance.neodymium.testclasses.multibrowser.BrowserstackHomePageTest;
+
+public class BrowserstackTest extends NeodymiumTest
+{
+    @BeforeClass
+    public static void beforeClass() throws IOException
+    {
+        Map<String, String> properties1 = new HashMap<>();
+        properties1.put("browserprofile.testEnvironment.browserstack.url", "https://hub-cloud.browserstack.com/wd/hub");
+        properties1.put("browserprofile.testEnvironment.browserstack.username", System.getenv("BROWSERSTACK_USERNAME"));
```

Comment:
Let's create an Owner configuration that can provide all information that is needed for testing purposes. By doing so we can have a local `config/dev-testconfiguration.properties` file to run the tests without setting up those values as environment variables. Please see if we already have a similar approach at another place and combine those to a general solution.

Reply:
Done